pkg/storaged/utils.js

```diff
@@ -682,16 +790,34 @@
             return actions;
         }
 
+        function enter_unmount(block, location, is_top) {
+            for (const u of usage) {
+                if (u.usage == 'mounted' && u.location == location) {
+                    if (is_top) {
+                        u.actions = get_actions(_("unmount"));
+                        u.set_noauto = false;
+                    }
+                    return;
+                }
+            }
+            usage.push({
+                level,
+                block,
+                usage: 'mounted',
+                location,
+                set_noauto: !is_top,
+                actions: is_top ? get_actions(_("unmount")) : [_("unmount")],
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19388-20230926-135906-360ec0bd-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)